src/earn/EarnEntrypoint.test.tsx

```diff
@@ -1,47 +1,148 @@
 import { fireEvent, render } from '@testing-library/react-native'
 import React from 'react'
+import { Provider } from 'react-redux'
 import AppAnalytics from 'src/analytics/AppAnalytics'
 import { EarnEvents } from 'src/analytics/Events'
 import EarnEntrypoint from 'src/earn/EarnEntrypoint'
+import { EarnTabType } from 'src/earn/types'
 import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
 import { getFeatureGate } from 'src/statsig'
 import { StatsigFeatureGates } from 'src/statsig/types'
+import { NetworkId } from 'src/transactions/types'
+import { createMockStore } from 'test/utils'
 
 jest.mock('src/statsig')
 
+function getStore(
+  { hasSuppliedPools }: { hasSuppliedPools: boolean } = { hasSuppliedPools: false }
```

Comment:
slightly less verbose
```suggestion
  { hasSuppliedPools = false }: { hasSuppliedPools: boolean } = {}
```